fabbwled-frontend/src/DiceComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import {useRef} from 'react';
+import ReactDice, {ReactDiceRef} from "react-dice-complete";
+
+interface DiceComponentProps {
```

Comment:
```suggestion
type DiceComponentProps {
```
just to streamline how we declare props